content/2-6.md

```diff
@@ -1,42 +1,43 @@
 ---
 title: "[Draft] Module 6: Custom Widgets"
-permalink: /curricula/developing-accessible-content/custom-widgets/
-ref: /curricula/developing-accessible-content/custom-widgets/
+permalink: /curricula/developer-modules/custom-widgets/
+ref: /curricula/developer-modules/custom-widgets/
 lang: en
 github:
   repository: w3c/wai-curricula
   path: content/2-6.md
 license: creative-commons
+acknowledgements: /curricula/acknowledgements/
 footer: >
   <p><strong>Date:</strong> $Date</p>
-  <p><strong>Editors: </strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p><strong>Editors:</strong> Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. ACKNOWLEDGEMENTS lists contributors and credits.</p>
   <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed with support from the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
 navigation:
-  previous: /curricula/developing-accessible-content/forms-and-input-elements/
+  previous: /curricula/developer-modules/forms/
+  next: /curricula/developer-modules/rich-applications/
 ---
 
 ## Introduction
 {:.no-display}
 
-Courses based on this module:
+Courses based on this module should:
 
-* explain use of proper coding for accessible custom widgets, including buttons, carousels, sliders, tabs, treeviews, and others defined in the WAI-ARIA authoring practices, section [Design patterns and widgets](https://www.w3.org/TR/wai-aria-practices/#aria_ex)
-* demonstrate how people with disabilities use accessibility properties, states, keyboard and focus interactions, and live regions to operate custom widgets
+* demonstrate how people with disabilities use properties, states, keyboard and focus interactions, and live regions to interact with custom widgets
+* explain coding techniques to create accessible custom widgets, including buttons, carousels, sliders, tabs, treeviews, and others defined in the WAI-ARIA Authoring Practices, section [Design patterns and widgets](https://www.w3.org/TR/wai-aria-practices/#aria_ex)
 
 ## Learning Outcomes for Module
 
 Students should be able to:
```

Comment:
I wonder if there needs to be something that reinforces the need to use native elements. For example:

> explain why native elements are preferable to custom elements

Reply:
This learning outcome is reflected in topic "Role Definitions" but could be reflected here too in a more generic form. Indeed, in the previous version we had "identify standard and custom widgets based on accessibility semantics" as a learning outcome, which somewhat covered this aspect. I think the suggestion from @iadawn is succinct and to the point.

Reply:
Added.